setup/install_gatling/30_post_install/50_configure_permissive_server_auth.rb

```diff
@@ -0,0 +1,39 @@
+test_name 'Configure permissive auth on Puppet Server'
+
+step 'Configure permissive auth.conf on master' do
+  supports_tk_auth = ENV['PUPPET_SERVER_TK_AUTH'] == 'true'
+
+  if supports_tk_auth
+    Beaker::Log.notify "Server supports tk auth, configuring."
+
+    auth_conf = '/etc/puppetlabs/puppetserver/conf.d/auth.conf'
+    create_remote_file(master, auth_conf, <<-EOF)
```

Comment:
In my gut, I think we should at least back up the original file, but this is essentially code from other parts of the project I am willing to :+1: now and submit a PR later.

Reply:
Looks like we are restarting puppet server in https://github.com/puppetlabs/gatling-puppet-load-test/pull/141/files#diff-5c5ae9d30fb908b001bb997c74792edb but as this comes before this test, the auth.conf changes still won't take effect.

Reply:
Good call, I re-named the tests to run in a different order.